### DISCUSSION TITLE
For You feed

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#355] [Announcements]
We just upgraded your experience!

Our documentation has been completely removed, and replaced with a **For You** feed.
It'll now be easier to find the *stories that you love* right here on GitHub.

As always, please leave your feedback below 👇

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/releases/tag/vyesterday'>For You feed</a>.</em>

**Comment by aslilac** [comment]
balls

**Comment by GoldenStack** [comment]
Does the 'for you' feed have phone integration so that it can learn what I like based off of everything that I say and hear during the day (without my consent, of course)?

**Reply by TodePond** [reply]
Yes! And if it's not to your preferences, just submit a PR to change up the feed.

**Reply by GoldenStack** [reply]
No I prefer to complain about it instead

**Comment by mybearworld** [comment]
This is the best GitHub repo I've ever seen

**Comment by 88572** [comment]
It should be the For You Page so that we can call it the FYP.

**Reply by mybearworld** [reply]
Everyone calls it FYP anyways :P

**Comment by callmewoem** [comment]
where can I

**Reply by mybearworld** [reply]
you can in

**Reply by loglot** [reply]
can i do?

**Reply by mybearworld** [reply]
you can do, but only when

**Reply by loglot** [reply]
Thank you very

**Reply by callmewoem** [reply]
you're

**Reply by loglot** [reply]
... you didn't

**Reply by callmewoem** [reply]
didn't i when can?

**Reply by mybearworld** [reply]
you didn't because doesn't

**Reply by loglot** [reply]
Yes, you

**Reply by mybearworld** [reply]
no but did?

**Comment by UserUNP** [comment]
Honestly quite incredible
